gromacs/tools.py

```diff
@@ -59,27 +59,13 @@
 from __future__ import absolute_import
 __docformat__ = "restructuredtext en"
 
-import os.path
-import tempfile
+import os.path, tempfile, subprocess, atexit, warnings
```

Comment:
PEP8 style is [one line per import](https://www.python.org/dev/peps/pep-0008/#imports), please adhere to it.